apps/web/pages/event-types/[type].tsx

```diff
@@ -366,6 +369,10 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
     minimumBookingNotice: number;
     beforeBufferTime: number;
     afterBufferTime: number;
+    seatsPerTimeSlot: number | null;
+    seatsPerAttendee: number;
```

Comment:
Not adding this feature in yet

Reply:
I think we can remove it or have it commented with a `// @NOTE:`